src/dimension.h

```diff
@@ -77,16 +77,28 @@ typedef struct Point
 	(USECS_PER_DAY) /* 1 day with adaptive                                                         \
 					 * chunking enabled */
 
+/* Default intervals for integer types */
+#define DEFAULT_SMALLINT_INTERVAL 10000
```

Comment:
Are these values based on an obversion of a certain workload or just default values?

Reply:
This is not based on any particular measurement.